specs/core/0_beacon-chain.md

```diff
@@ -216,6 +216,8 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_EPOCHS_PER_CROSSLINK` | `Epoch(2**6)` (= 64) | ~7 hours |
 | `MIN_EPOCHS_TO_INACTIVITY_PENALTY` | `Epoch(2**2)` (= 4) | 25.6 minutes |
 
+- `MAX_EPOCHS_PER_CROSSLINK` should be a small constant times `SHARD_COUNT // SLOTS_PER_EPOCH`.
```

Comment:
Design rationales are not included in the spec. I added this line in https://github.com/ethereum/eth2.0-specs/issues/407

Reply:
Ah, it seemed natural in the conflict to merge. But will amend it, and remove it.

Reply:
removed it in amended merge commit ab2001e